test/train-sets/ref/help_cbadf.stdout

```diff
@@ -121,6 +121,8 @@ Input Options:
                                             keep)
     --flatbuffer                            Data file will be interpreted as a flatbuffer file (type: bool,
                                             experimental)
+    --api_status                            api_status object will not be set to nullptr which will show
```

Comment:
--detailed_errors, --error_info, --error_details?

Reply:
I'm kind of tempted to remove the flag and always send it in inside the call. Since the errors lead to a throw and parse terminate, not really getting a big performance boost out of avoiding the string constructions on error.